index.bs

```diff
@@ -945,9 +951,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 : {{AuthenticatorAssertionResponse/signature}}
                 :: A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of the returned
                     {{signature}}
-                : {{AuthenticatorAssertionResponse/userId}}
-                :: A new {{DOMString}} containing the user handle returned from the successful [=authenticatorGetAssertion=] operation, 
-                    as defined in [[#op-get-assertion]].   
+                : {{AuthenticatorAssertionResponse/userHandle}}
+                :: A new {{ArrayBuffer}} containing the [=user handle=] returned from the successful [=authenticatorGetAssertion=]
```

Comment:
Add `, created using |global|'s [=%ArrayBuffer%=],` like in the earlier items.

(I wish we had a more concise way to say this, but this is the state of the art at the moment.)

Reply:
Fixed!